test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ApplicationInsights/ServiceBusRequestAndDependencyCollectionTests.cs

```diff
@@ -67,16 +66,27 @@ await host.GetJobHost()
             Assert.Equal(2, requests.Count);
             Assert.Single(dependencies);
 
-            Assert.Single(requests.Where(r => r.Context.Operation.ParentId == dependencies.Single().Id));
-            var sbTriggerRequest = requests.Single(r => r.Context.Operation.ParentId == dependencies.Single().Id);
+            Assert.Single(requests.Where(r => r.Name == nameof(ServiceBusTrigger)));
+            var sbTriggerRequest = requests.Single(r => r.Name == nameof(ServiceBusTrigger));
             var manualCallRequest = requests.Single(r => r.Name == nameof(ServiceBusOut));
             var sbOutDependency = dependencies.Single();
 
-            string operationId = manualCallRequest.Context.Operation.Id;
-            Assert.Equal(operationId, sbTriggerRequest.Context.Operation.Id);
+            string manualOperationId = manualCallRequest.Context.Operation.Id;
+            string triggerOperationId = sbTriggerRequest.Context.Operation.Id;
 
-            ValidateServiceBusDependency(sbOutDependency, _endpoint, _queueName, "Send", nameof(ServiceBusOut), operationId, manualCallRequest.Id);
-            ValidateServiceBusRequest(sbTriggerRequest, _endpoint, _queueName, nameof(ServiceBusTrigger), operationId, sbOutDependency.Id);
+            // currently ApplicationInsights supports 2 parallel correlation schemes:
+            // legacy and W3C, they both appear in telemetry. UX handles all differences in operation Ids. 
+            // This will be resolved in next .NET SDK on Activity level
+            string dependencyLegacyId = sbOutDependency.Properties.Single(p => p.Key == "ai_legacyRequestId").Value;
+            string triggerCallLegacyRootId = sbTriggerRequest.Properties.Single(p => p.Key == "ai_legacyRootId").Value;
+            string manualCallLegacyRootId = manualCallRequest.Properties.Single(p => p.Key == "ai_legacyRootId").Value;
```

Comment:
These are validating the legacy correllation? Do we validate W3C anywhere?

Reply:
we do in http tests. We also test W3C extensively in AI SDK.

Also, this is ServiceBus tests and ServiceBus does not implement W3C just yet, but below in the same test there is tricky correlation validation. 

For the end users UX handles all this tricky things and end users should not notice there are 2 separate correlation schemas involved.